client/me/purchases/manage-purchase/index.tsx

```diff
@@ -1512,6 +1482,90 @@ function addPaymentMethodLinkText( {
 	return linkText;
 }
 
+function BBEPurchaseDescription( { purchase }: { purchase: Purchase } ) {
+	const translate = useTranslate();
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, purchase.siteId ) );
+	const { isLoading, data: websiteContentQueryResult } = useGetWebsiteContentQuery( siteSlug );
+	const difmTieredPurchaseDetails = getDIFMTieredPurchaseDetails( purchase );
+	if ( ! difmTieredPurchaseDetails ) {
+		return;
+	}
+
+	const extraPageCount = difmTieredPurchaseDetails.extraPageCount || 0;
+	const numberOfIncludedPages = difmTieredPurchaseDetails.numberOfIncludedPages as number;
+
+	const BBESupportLink = (
+		<a
+			href={ `mailto:builtby+express@wordpress.com?subject=${ encodeURIComponent(
```

Comment:
Do we normally use mailto? We could perhaps take advantage of our HelpCenter implementation and redirect through the help center form to the appropriate ZD queue:

![image](https://github.com/Automattic/wp-calypso/assets/16580129/4d0f2cc8-f1be-406f-acad-3f219476d50a)

Reply:
We do use mailto in one other place: https://github.com/Automattic/wp-calypso/blob/df05ccf161f12e95422f0c807ff04a241bc99bf9/client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx#L61

But you're absolutely right, the mailto UX is less than ideal, and we should switch to the HelpCenter. I'm merging this for now, and will tackle this in a follow-up PR.